es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="my-5 container">
```

Comment:
Ah this is one change from the version of this page on `es-site` - we use bootstrap-vue, which means instead of `class="container"` the div will be a `<b-container>` component.  Also will replace BContainer, BRow, and BCol, [documentation here](https://bootstrap-vue.org/docs/components/layout)